l10n.toml

```diff
@@ -0,0 +1,87 @@
+
+basepath = "."
```

Comment:
nit: Does this need a license? Also, blank line above is probably unnecessary.

Reply:
I'll add one, and keep the newline to separate that ;-)